site/src/pages/RoadmapPage/transfers/UncategorizedCreditsSection.tsx

```diff
@@ -6,19 +6,33 @@ import { removeUncategorizedCourse } from '../../../store/slices/transferCredits
 import { useAppDispatch, useAppSelector } from '../../../store/hooks';
 import { TransferredUncategorized } from '@peterportal/types';
 
-const UncategorizedMenuTile: FC<TransferredUncategorized> = ({ name, units }) => {
+const UncategorizedMenuTile: FC<{ uncategorizedInfo: TransferredUncategorized; unread: boolean }> = ({
+  uncategorizedInfo,
+  unread,
+}) => {
```

Comment:
Following the previous comment, I'd like to still be able to read name and units:

```suggestion
const UncategorizedMenuTile: FC<...> = ({ name, units, unread }) => {
```
The `FC`'s type could be a `type UncategorizedMenuTileProps = TransferredUncategorized & { unread; }`, but if you also take the suggestion of creating a frontend type that adds `unread?: boolean` on top of the transfers, then you can directly use that type, i.e. `FC<TransferWithUnread<TransferredUncategorized>>`.

(you could also name it something like `DraftTransfer` idk, this depends on how much abstraction you like)